aptos-move/framework/supra-framework/sources/block.move

```diff
@@ -235,6 +236,7 @@ module supra_framework::block {
         randomness::on_new_block(&vm, epoch, round, option::none());
         if (timestamp - reconfiguration::last_reconfiguration_time() >= epoch_interval) {
             reconfiguration::reconfigure();
+            automation_registry_state::on_new_epoch(epoch_interval);
```

Comment:
This should be called from `reconfiguration`

Reply:
reconfiguration does not have access to epoch-interval, either `reconfiguration::reconfigure()` should be updated to accept epoch_interval as parameter or we should leave it as it is.